scripts/ci/prek/check_common_compat_lazy_imports.py

```diff
@@ -17,16 +17,15 @@
 # under the License.
 
 """
-Check and generate sdk.pyi from sdk.py.
+Validate that TYPE_CHECKING block in sdk.py matches runtime import maps.
```

Comment:
Just changing from separate `pyi` file to `TYPE_CHECKING` block within same file  `sdk.py`